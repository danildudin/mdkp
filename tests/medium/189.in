28 9 0
53 249 739 664 515 507 928 851 380 788 965 970 153 712 81 481 10 254 787 216 622 249 288 502 107 264 787 232 
147 652 100 311 726 620 91 891 927 494 707 309 920 603 409 229 298 254 638 135 36 1000 713 222 507 131 227 42 
783 698 546 298 798 447 582 991 136 499 224 122 722 853 758 931 873 689 196 308 542 496 869 418 147 489 163 279 
204 656 883 382 367 969 608 623 955 464 829 857 281 765 921 666 18 555 818 279 180 212 627 544 773 175 709 742 
677 133 262 984 467 470 579 223 581 743 99 181 380 216 533 13 893 201 482 127 716 670 633 22 293 866 994 939 
101 246 573 613 80 360 778 881 603 118 756 512 231 505 191 543 408 228 621 681 746 685 526 499 988 789 121 529 
348 492 519 965 472 769 990 675 770 792 128 744 751 276 325 761 464 468 193 690 313 104 288 436 564 901 794 570 
976 655 504 990 363 634 263 685 282 94 945 727 590 263 986 940 930 565 843 150 497 711 662 716 412 561 260 341 
42 671 10 812 348 833 600 442 1000 814 114 186 358 607 575 824 194 739 653 629 826 847 939 250 682 767 886 514 
82 846 639 892 285 102 5 195 165 123 763 109 341 189 1 274 746 655 398 378 692 401 683 856 745 945 902 317 
5627 7258 5081 6620 5010 5562 6649 5404 6839 
