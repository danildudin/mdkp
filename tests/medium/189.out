7015
9
3 4 6 7 9 10 11 13 20 
